.gitignore

```diff
@@ -2,4 +2,5 @@
 .DS_Store
 bin/configlet
 bin/configlet.exe
+elm-stuff
```

Comment:
Oddly this is where Elm puts dependencies and compiler crap. Seems that everyone will have their own. I would think this is the right place.